File: ch1/ch1-pg7-exam2.cpp
#include <stdio.h>

int main()
{
    int num;
    int firstD, secondD, thirdD;
    scanf("%d", &num);

    firstD = num / 100;
    secondD = num % 100 / 10;
    thirdD = num % 10;

    printf("%03d\n", thirdD * 100 + secondD * 10 + firstD);

    return 0;
}

File: ch1/ch1-pg7-exam2.js
const lnrd = require("readline");
const cslLnrd = lnrd.createInterface({
    input:process.stdin
});
cslLnrd.on("close", () => {
    process.exit();
});

let num;
let firstD;
let secondD;
let thirdD;

cslLnrd.on("line", input => {
    num = parseInt(input);
    firstD = Math.floor(num / 100);
    secondD = Math.floor(num % 100 / 10);
    thirdD = num % 10;

    console.log("The reverse vertion of 3digit number input is: ", thirdD * 100 + secondD * 10 + firstD);
    cslLnrd.close();
});
